README.md

```diff
@@ -52,13 +55,10 @@ The table below presents the mutable fields of the contract and their initial va
 |`owner`| `ByStr20` | `init_owner` | Current `owner` of the contract. |
 |`pauser`| `ByStr20` | `init_owner` | Current `pauser` in the contract. |
 |`masterMinter`| `ByStr20` | `init_owner` | Current `masterMinter` in the contract.|
-|`blacklister`| `ByStr20` | `init_owner` | Current `blacklister` in the contract.|
 |`paused`| `Bool` | `True` | Keeps track of whether the contract is current paused or not. `True` means the contract is paused. |
+|`blacklister`| `ByStr20` | `init_owner` | Current `blacklister` in the contract.|
 |`blacklisted`| `Map ByStr20 Uint128` | `Emp ByStr20 Uint128` | Records the addresses that are blacklisted. An address that is present in the map is blacklisted irrespective of the value it is mapped to. |
-|`balances`| `Map ByStr20 Uint128` | `Emp ByStr20 Uint128` | Keeps track of the number of tokens that each token holder owns. |
-|`allowed`| `Map ByStr20 (Map ByStr20 Uint128)` | `Emp ByStr20 (Map ByStr20 Uint128)` | Keeps track of the `approvedSpender` for each token holder and the number of tokens that she is allowed to spend on behalf of the token holder. |
-|`totalSupply`| `Uint128`| `0` | The total number of tokens that is in the supply. |
-|`minters`| `Map ByStr20 Uint128`| `Emp ByStr20 Uint128` | Maintains the current `minter`s. An address that is present in the map is a `minter` irrespective of the value it is mapped to.|
+|`allowed`| `Map ByStr20 (Map ByStr20 Uint128)` | `Emp ByStr20 (Map ByStr20 Uint128)` | Keeps track of the `Spender` for each token holder and the number of tokens that she is allowed to spend on behalf of the token holder. |
 |`minterAllowed`| `Map ByStr20 Uint128` | `Emp ByStr20 Uint128` | Keeps track of the allowed number of tokens that a `minter` can mint. |
```

Comment:
This needs to be updated to be Option, and elaborate a little on the rationale behind the `Option`